### ISSUE TITLE
Implement Credits Activity

### ISSUE THREAD

**Issue by foxtrotzulu94** [foxtrot94/McHacksPokedex#28]
As a form of correctly attributing the work done by us (all developers) and the creators of the Pokemon Franchise, create an Activity which lists the developers names, the companies behind Nintendo and mentions the many fan-supported websites used to make this project possible.

**Comment by minicole** [comment]
Should there be a link here to ask people to rate the product? Or should we make a separate issue to add a pop up after X hours of use to ask them to rate it? Nothing too annoying, just a little popup that they can just ignore if they want...

**Comment by foxtrotzulu94** [comment]
Yeah, we could do both. For now we can put that link ever so subtly there and then, in a separate issue, program the pop-up behaviour after, say, 2 hours?

**Comment by foxtrotzulu94** [comment]
I will take a look into this as well.
